agent.py

```diff
@@ -895,7 +893,12 @@ def findNeighborhood(self, newCell=None):
                 neighborhood.append(neighbor)
         neighborhood.append(self)
         if newCell == None:
+            # Shuffle cells for movement considerations
+            random.shuffle(allCells)
```

Comment:
Double checking: this shuffle that was moved down from `findCellsInRange` should be simply removed, correct?

Reply:
This shuffle will be called with the empty parameter findNeighborhood(), which is called at the top of findBestCell() and will update self.neighborhood. I figured you would want to have self.neighborhood be shuffled, but it doesn't actually seem like it's used anywhere in the code except for ethical calculations in ethics.py where it doesn't need to be shuffled. Looks like the shuffle() can be completely removed.

Reply:
Correct, the shuffle is not needed at all. When `findNeighborhood` is called without parameters, the shuffle we actually care about happens two lines after the call.